### ISSUE TITLE
Checkra1n error -76 on Samsung Galaxy A12

### ISSUE THREAD

**Issue by ghost** [checkra1n/BugTracker#2202]
**Tell us about your setup:**
1. What iDevice are you using?
iPhone 8 (A1905), board config: D201AP
2. On what version of iOS is it?
14.7.1 (18G82)
3. What version of checkra1n are you using?
checkra1n 0.12.4 (latest) arm64
4. What is your host system (OS version? Hackintosh? VM? etc.)?
Android 11 (linux kernel version: 4.19.127-22457755); also tried in TWRP 3.5.2_11
5. How are you connecting to the device (USB-A? USB-C? Apple/3rd party cable? Through a USB hub?)?
USB-C to USB-A OTG adapter and a lightning cable (cable confirmed to work)
**What are the steps to reproduce the issue?**
1. Run checkra1n 0.12.4 arm64 on a rooted galaxy A12
**What do you expect, and what is happening instead?**
I expect the iPhone to be jailbroken, I get error -76 instead

**Did you see a popup on the device stating it entered "Safe Mode" due to an error?**
no

**Does the issue also occur if you tick "Safe Mode" in the checkra1n options?**
yes

**Any other info, error logs, screenshots, ...?**
![checkra1n](https://user-images.githubusercontent.com/49041196/139538457-b8f1d6c3-92b2-474f-8d36-1fe1fe30f712.jpeg)
![checkra1n2](https://user-images.githubusercontent.com/49041196/139538500-8ff6918e-4a7d-43ee-8476-f88e924720e7.jpeg)

**Comment by feljos** [comment]
What do you expect. Checkra1n running on Android? Can believe that

**Comment by ghost** [comment]
Yes android is based on the linux kernel and checkra1n offers support for any os that is based on linux kernel version 4 or later so this is one of the supported operating systems.

**Comment by feljos** [comment]
Never know that. So what is the use of installing checkra1n on android?

**Comment by feljos** [comment]
Is checkra1n offer you a terminal installation after rooting an android?

**Comment by ghost** [comment]
I tried it just to see if it is possible. It would also be useful to have a checkra1n jailbreak machine with me all the times so i do not need to carry a computer.

**Comment by ghost** [comment]
It is termux, an android play store application.

**Comment by ghost** [comment]
Also if you did not understand it correctly i am not rooting an android with checkra1n, i am using a rooted phone to jailbreak an iphone with checkra1n.

**Comment by feljos** [comment]
You're trying to telling me that like if you want to root an android, you directly use checkra1n for it and it'll works?

**Comment by ghost** [comment]
You can not root an android with checkra1n, you can use checkra1n on a rooted phone to jailbreak iPhones.

**Comment by feljos** [comment]
So if you have a rooted android and you install checkra1n on it. You'll use it to jailbreak a iPhone using a OTG cable connect iPhone?

**Comment by ghost** [comment]
Yes correct.

**Comment by feljos** [comment]
Thanks so much.

**Comment by feljos** [comment]
But I don't still understand one thing. Termux also works on it

**Comment by ghost** [comment]
What do you mean

**Comment by feljos** [comment]
Because I see that command line on the screenshot u post. Is that checkra1n running or termux

**Comment by ghost** [comment]
That is checkra1n running in termux. Termux is a terminal not a jailbreak tool.

**Comment by feljos** [comment]
Thanks a lot

**Comment by feljos** [comment]
It works with any android?

**Comment by ghost** [comment]
It should work with any android but support is only offered for android versions with  linux kernel 4 or later.

**Comment by feljos** [comment]
Ok thank you so much for helping me
